docs/t-sql/database-console-commands/dbcc-checkident-transact-sql.md

```diff
@@ -160,15 +160,19 @@ GO
 
 ### D. Resetting the identity value on an empty table
 
- The following example forces the current identity value in the `ErrorLogID` column in the `ErrorLog` table to a value of 1, after deleting all records from table. Because the table has no existing rows, the next row inserted will use 1 as the value, that is, the new current identity value, without adding the increment value defined for the column.  
+ The following example assumes a table identity of `(1,1)` and forces the current identity value in the `ErrorLogID` column in the `ErrorLog` table to a value of 1, after deleting all records from table. Because the table has no existing rows, the next row inserted will use 1 as the value. That is, the new current identity value without adding the increment value defined for the column after TRUNCATE, or adding the increment value after DELETE.  
```

Comment:
The grammar didn’t work with “That is” starting on a new sentence, so I suggest switching back to a comma.

```suggestion
 The following example assumes a table identity of `(1,1)` and forces the current identity value in the `ErrorLogID` column in the `ErrorLog` table to a value of 1, after deleting all records from table. Because the table has no existing rows, the next row inserted will use 1 as the value, that is, the new current identity value without adding the increment value defined for the column after TRUNCATE, or adding the increment value after DELETE.  
```